src/components/appLayout.js

```diff
@@ -51,7 +51,7 @@ class AppLayout extends React.Component {
   render() {
     const { psc, account, web3, section, bridge, bridgeAddr } = this.props;
 
-    if (!account.ready) {
+    if (window.web3 && !account.ready) {
```

Comment:
this is the change. The rest is reformat by precommit hook + moving Web3Warning 🤷‍♂️

Reply:
`account.ready` should have a proper value without additional condition here

Reply:
changed a bit. I want to show the spinner only when web3 is injected, but accounts are not yet ready. If there is no web3 injected (and `account.ready` will be false of course), I want to show the app.

Like this:

web3 injected | web3 ready | accounts ready | outcome
--- | --- | --- | ---
| 🛑 |  |  | show the app with warning |
| ✅ |  |  | don't show the app, show loading spinner only |
| ✅ | ✅ |  | don't show the app, show loading spinner only |
| ✅ | ✅ | ✅ | show the app |